pkg/pillar/types/verifiertypes.go

```diff
@@ -119,7 +119,8 @@ func (status VerifyImageStatus) ImageDownloadFilenames() (string, string, string
 		// Else..VMs
 		pendingFilename = pendingDirname + "/" + status.Safename
 		verifierFilename = verifierDirname + "/" + status.Safename
-		verifiedFilename = verifiedDirname + "/" + status.Safename
+		filename := SafenameToFilename(status.Safename)
+		verifiedFilename = verifiedDirname + "/" + filename
```

Comment:
Is this an unrelated bugfix??

Reply:
I think we were just lucky we did not hit this one.. Once I removed the dedicated code in Verifier to validate during bootup, the bug was uncovered. The previous code had its own hard-coded logic for the verified image location. Now the bug in this routine got exposed.

Reply:
OK